test/e2e-seed-data-framework/.env.local

```diff
@@ -0,0 +1,14 @@
+# General
+IDS_START_FROM=1000000 # All generated test IDs start from 1,000,000
+
+# FamilyHubs.ServiceDirectory.Database
+DB_SD_HOST=localhost # Database host
```

Comment:
Do we want to constrain users to sql user/pwd? If running locally, could use integrated security. Also if we decide to proceed with the Entra work, the users will need to be Entra users. Maybe a connection string is better?

Reply:
Good shout. Will look to support connection strings.

Reply:
Resolved: [efca615](https://github.com/DFE-Digital/fh-services/pull/171/commits/efca6153afd4eaea42ff4d32cddbf2d4e5113377) and [ca6934c](https://github.com/DFE-Digital/fh-services/pull/171/commits/ca6934c232c6051c387b5e9ae6db3a3f69fd57ad)